pkgs/by-name/va/vaults/package.nix

```
@@ -0,0 +1,86 @@
+{ fetchFromGitHub
+, lib
+, stdenv
+, appstream-glib
+, desktop-file-utils
+, meson
+, ninja
+, pkg-config
+, python3
+, rustPlatform
+, rustc
+, cargo
+, wrapGAppsHook
+, glib
+, gtk4
+, libadwaita
+, wayland
+, gocryptfs
+, cryfs
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vaults";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "mpobaschnig";
+    repo = "Vaults";
+    rev = version;
+    fetchSubmodules = false;
+    hash = "sha256-x7NoYQ+ol/j8LMazL4A0jDi/I4ajRNUzOpShLB0eHUg=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    sha256 = "sha256-fDUsbeJzgqjvNCNce1FnvvqZfXu0X5Knpan4Q+PYe+Q=";
+  };
+
+  postPatch = ''
+    patchShebangs build-aux
+  '';
+
+  makeFlags = [
+    "PREFIX=${placeholder "out"}"
+  ];
+
+  preFixup = ''
+    gappsWrapperArgs+=(
+      --prefix PATH : "${lib.makeBinPath [ gocryptfs cryfs ]}"
+    )
+  '';
+
+  nativeBuildInputs = [
+    appstream-glib
+    gtk4
+    desktop-file-utils
+    meson
+    ninja
+    cmake
+    pkg-config
+    python3
+    wrapGAppsHook
+    cargo
+    rustc
+    rustPlatform.cargoSetupHook
+  ];
```

```suggestion
  nativeBuildInputs = [
    desktop-file-utils
    meson
    ninja
    cmake
    pkg-config
    wrapGAppsHook
    cargo
    rustc
    rustPlatform.cargoSetupHook
  ];
```
The removed items only need to be in `buildInputs`